test/cmdlineTests/external_function_pointers_taking_calldata_args/input.json

```
@@ -0,0 +1,13 @@
+{
+    "language": "Solidity",
+    "sources": {
+        "url_not_found.sol": {
+            "urls": [
+                "/projects/contracts.sol",
+                "contract.sol",
```

This is not the correct path. In external tests the working dir is `test/cmdlineTests/`. So you need to add the test dir name as well:
```suggestion
                "external_function_pointers_taking_calldata_args/contract.sol",
```

This is why you're getting `IOError`s for all three files.